example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -26,6 +26,7 @@ controllers:
   # enableShootCoreAddonRestarter: true
   shootHibernation:
     concurrentSyncs: 5
+  # triggerDeadlineDuration: 30s
```

Comment:
Please add this as well to:

- https://github.com/gardener/gardener/blob/471f9184f35fda756b267badb039659c6a2314e5/charts/gardener/controlplane/values.yaml#L399-L401
- https://github.com/gardener/gardener/blob/471f9184f35fda756b267badb039659c6a2314e5/charts/gardener/controlplane/charts/runtime/templates/controller-manager/configmap-componentconfig.yaml#L117-L118 (btw the `syncPeriod` was not made configurable here, so you can do this as well)